### ISSUE TITLE
Error updating kube-proxy 1.20 to 1.21

### ISSUE THREAD

**Issue by YonierGomez** [weaveworks/eksctl#4041]
### Cordial greeting

I report the following case:
Currently we have an error when updating kube-proxy with eksctl going from version 1.20 to version 1.21, the specific error is the following:

```bash
$ eksctl utils update-kube-proxy --cluster=<cluster-name> --approve
``` 

`###  Warning  InspectFailed  4m23s (x26 over 9m28s)  kubelet            Failed to apply default image tag "602401143452.dkr.ecr.us-east-1.amazonaws.com/eks/kube-proxy:v1.21.2+d2965f0db10712-eksbuild.1": couldn't parse image reference "602401143452.dkr.ecr.us-east-1.amazonaws.com/eks/kube-proxy:v1.21.2+d2965f0db10712-eksbuild.1": invalid reference format`

> This error is reflected in all clusters when executing the update

```
My version of eksctl is the: 0.59.0
eks version: 1.21
```

**Comment by hikhvar** [comment]
I see this in my cluster also.

**Comment by hikhvar** [comment]
I think this is solved in: #4021
